plugin-statusnotifier/statusnotifierbutton.cpp

```diff
@@ -70,6 +71,12 @@ StatusNotifierButton::StatusNotifierButton(QString service, QString objectPath,
     connect(interface, &SniAsync::NewToolTip, this, &StatusNotifierButton::newToolTip);
     connect(interface, &SniAsync::NewStatus, this, &StatusNotifierButton::newStatus);
 
+    // get the title only at the start
+    interface->propertyGetAsync(QLatin1String("Title"), [this] (QString value) {
+        mTitle = value;
+        emit titleFound(value);
```

Comment:
I consider `Q_EMIT` better than `emit`. Though no official announcements, KDE devs said that `QT_NO_KEYWORDS` will be default in Qt 6 [1].

[1] https://github.com/KDE/kalarm/commit/29eed84d11371a41314282abb46ab6cc247e153b

Reply:
Thanks! Will change that.